rust/src/database/verkle/variants/managed/commitment.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2025 Sonic Operations Ltd
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at soniclabs.com/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+use zerocopy::{FromBytes, Immutable, IntoBytes, Unaligned};
+
+use crate::{
+    database::{managed_trie::TrieCommitment, verkle::crypto::Commitment},
+    types::Value,
+};
+
+/// The commitment of a managed verkle trie node, together with metadata required to recompute
+/// it after the node has been modified.
+///
+/// NOTE: While this type is meant to be part of trie nodes, a dirty commitment should never
+/// be persisted to disk. The dirty flag is nevertheless part of the on-disk representation,
+/// so that the entire node can be transmuted to/from bytes using zerocopy.
+/// Related issue: https://github.com/0xsoniclabs/sonic-admin/issues/373
```

Comment:
I guess the same holds for `cahgned`. It should always be all 0 when persisted to disk, no?

Reply:
Yes, changed the comment.